Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install --upgrade --no-cache-dir black
```

Comment:
Why skip the cache? CIs can account for a lot of downloads from PyPI and it'd be good to minimise their huge hosting bills where possible.

Also, some may prefer to pin to a given version of Black (eg. how pre-commit does it)

Reply:
Because this run inside docker's build system, it runs on a clean `python:3.8` image each time. The cache from previous runs is never available.

The goal of `--no-cache-dir` here  it for docker not to _save_ the cache, since it ends up in the image otherwise, bloating it with a cache that will never be read.